### DISCUSSION TITLE
Suggestion: postgres default encoding should be UTF8 not ascii

### DISCUSSION THREAD

**Discussion by timdonovanuk** [tteck/Proxmox#1152] [General]
What year is it? :older_man:

The majority of applications these days prefer your database to be in utf-8. I do not know why Postgres still defaults to ASCII.  You have to migrate or fiddle around, e.g. 

- https://www.shubhamdipt.com/blog/how-to-change-postgresql-database-encoding-to-utf8/ 
- http://mysqltopgsql.com/post/unicode/ 

I recently ran into an issue with https://hay-kot.github.io/mealie/ because the data it was trying to store was UTF.

Just a suggestion to perhaps change the Postgres default in your lovely LXC to be UTF. Thanks!

**Comment by tteck** [comment]
Fixed https://github.com/tteck/Proxmox/commit/e6db5a353b349a203a5b61e3a6caa38ed6d39d13

**Reply by tteck** [reply]
Probably a good idea to ensure  all LXC's use UTF-8.

**Comment by timdonovanuk** [comment]
Awesome, thank you! <3
